modules/client-configuration/pages/clients-rh.adoc

```diff
@@ -6,36 +6,67 @@ This section contains information about registering traditional and Salt clients
 
 
 
-== Set up a {rhel} Client
+== Server requirements
 
-Ensure that your client meets these requirements before you start:
-
-* 8{nbsp}GB RAM or more
-* Two or more physical or virtual CPU cores
-* Access to {rhel} installation and subscription media
-* An LVM or an NFS mount is recommended
+[WARNING]
+====
+You are responsible for arranging access to Red Hat base media repositories and RHEL installation media.
+You must obtain support from either Red Hat or {suse} for all your RHEL systems.
```

Comment:
`or`? Shouldn't it be `and` as we request the user to use an original media, and then use RES?

Reply:
Seems so. Maybe say "You must obtain support from both Red Hat and {suse} for all your RHEL systems.:"

Reply:
I am not sure if `and` is correct. What do you want to say ?

Reply:
@mcalmer, that you need support from Red Hat (to be able to get RHEL installation media and have the right to use it) `AND` SUSE (to be able to get RES updates).

Isn't that the case?

Reply:
No. If People have official RHEL contract, they would not buy RES. RES is there to not need a subscription from RedHat.

RES was for "switching" to SUSE. The idea:

- You had a subscription with RedHat in the past
- From that times you have a media
- Now you switch to RES and use it for updates only.
- When you install a new server you use SLES and not RHEL.

(But this is theory, in practice people do not switch. But this is there problem)

Btw: A valid media os also "CentOS" for RES. And this is "free".

Reply:
I understand that, but then:
> You must obtain support from either Red Hat or {suse} for all your RHEL systems.

Why "Red Hat **OR** SUSE"? With your explanation, I'd say this warning is not needed.

> Btw: A valid media os also "CentOS" for RES. And this is "free".

Is this supported? If so, it should be mentioned at this manual, I guess.